pkg/registry/search/storage/proxy_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2023 The Karmada Authors.
```

Comment:
2023 -> 2024

The time refers to the time when the file is created.

Reply:
@XiShanYongYe-Chang hey , can you please review , the year thing has been fixed in latest commits